core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -9,24 +9,11 @@ import (
 
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+	decryptionPlugin "github.com/smartcontractkit/tdh2/go/ocr2/decryptionplugin"
```

Comment:
Do you specifically want to have a camelcase "decryptionPlugin"? If not you can remove that and just use the actual package name which is lowercase "decryptionplugin". There's no name collision.